### PR TITLE
Bugfixes - mpe channel handling and mono expression downgrading

### LINKED ISSUE #511: All NRPNs change MPE zone
### Please describe the problem:

Any NRPN received on channel 1 or 16 will change the number of MPE channels on the associated device

### What is the expected behavior?

They get ignored (since we can't handle NRPNs yet)

### Is there a relevant Pull request?

no

### What hardware did you reproduce it with?

OLED

### What firmware did you reproduce it with?

nightly

### What is the firmware name:

_No response_

### If possible provide the steps to reproduce the issue and upload additional media:

Open up midi MPE settings for a device
Send an RPN/NRPN from that device on channel 1 or 16
Observe the number of midi channels in that zone changing

### PR DESCRIPTION
**Pull request by m-m-adams**
Fix #511 

Also fix other bugs related to #506 - essentially channel 0 was being treated as MPE. The bug arises because if the last MPE channel is 0 there are no MPE channels, but if it is 1 then there are 2 (and opposite for ch 15). The existing code didn't handle this case and treated max MPE channel 0 as if it was a 1 channel MPE zone. These checks are now all done by the appropriate method from midi device. 

Also directs the monophonic expression event y axis to modwheel - this makes more sense than CC74 when collapsing an MPE channel to a single midi channel since most non MPE synths don't usefully use CC74. Similarly sends mod wheel to the y axis expression, allowing it to be mapped on the deluge

### CODE REVIEW

File: src/deluge/model/instrument/midi_instrument.cpp [hunk 1]
```diff
@@ -226,7 +226,8 @@ void MIDIInstrument::monophonicExpressionEvent(int32_t newValue, int32_t whichEx
 	}
 
 	case 1:
-		midiEngine.sendCC(masterChannel, 74, (newValue >> 25) + 64, channel);
+		//send CC1 for monophonic expression - monophonic synths won't do anything useful with CC74
+		midiEngine.sendCC(masterChannel, 1, (newValue >> 25) + 64, channel);
```
Comment (bobtwinkles) [inline src/deluge/model/instrument/midi_instrument.cpp:230]: this seems like it has potential to break setups, can/should we make it configurable?

Comment (m-m-adams) [reply src/deluge/model/instrument/midi_instrument.cpp:230]: That's a fair point. I don't know of a non MPE synth that uses CC74 as a general mod destination but they might exist

File: docs/community_features.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ Here is a list of general improvements that have been made, ordered from newest
 - ([#17]) Increase the resolution of "patch cables" between mod sources and destinations.
 
 #### 3.2 - MPE
-- ([#29]) Bugfix to respect MPE zones in kit rows. In the official firmware kit rows with midi learned to a channel would be triggered by an MPE zone which uses that channel. With this change they respect zones in the same way as synth and midi clips.
+- ([#29]) Bugfix to respect MPE zones in kit rows. In the official firmware kit rows with midi learned to a channel would be triggered by an MPE zone which uses that channel. With this change they respect zones in the same way as synth and midi clips. ([512]) adds further fixes related to channels 0 and 15 always getting received as MPE.
```
Comment (bobtwinkles) [inline docs/community_features.md:18]: `512` -> `#512`, and  you need to add an entry at the end of the document to the list of links

**Review by bobtwinkles — CHANGES_REQUESTED** [review]

File: src/deluge/model/instrument/midi_instrument.cpp (hunk 1 above)
Comment (bobtwinkles) [reply src/deluge/model/instrument/midi_instrument.cpp:230]: (discussed on Discord) - That seems fine, we can add a menu item for this in the future. Required for nymphes, I think.

**Review by bobtwinkles — APPROVED** [review]

**Pull request merged**